Update workspace-design.md (#82)
MicrosoftDocs/azure-monitor-docs: articles/azure-monitor/logs/workspace-design.md

### Hybrid
In a hybrid model, each tenant has its own workspace. A mechanism is used to pull data into a central location for reporting and analytics. This data could include a small number of data types or a summary of the activity, such as daily statistics.

There are several options to implement logs in a central location:

> - **Central workspace**: The service provider creates a workspace in its tenant and uses various techniques to pull data from the various workspaces:
>     - Script that utilizes the [Query API](api/overview.md) with the [logs ingestion API](logs-ingestion-api-overview.md) to bring the data from the tenant workspaces to this central location.
>     - Using [Azure Logic Apps](/azure/logic-apps/logic-apps-overview) to copy data to the central workspace.
>     - [Exporting data](logs-data-export.md) from the source workspace and re-ingest to the central workspae. When used together with [Summary Rules](summary-rules.md), it allows exporting aggregation of the original workspaces in the central workspaces.

Reviewer: ```suggestion
- **Central workspace**: The service provider creates a workspace in its tenant and pulls data from the various workspaces using:
    - A script that uses the [Query API](api/overview.md) with the [logs ingestion API](logs-ingestion-api-overview.md) to send the data from the tenant workspaces to the central workspace.
    - [Azure Logic Apps](/azure/logic-apps/logic-apps-overview) to copy data to the central workspace.
    - [Data export](logs-data-export.md) from the source workspace and re-ingestion to the central workspace. You can also create [summary rules](summary-rules.md) to export an aggregation of key data from the original workspaces into the central workspace.
```
